### PR TITLE
Allow adding new MediaContainerProbes via plugins

### PR DESCRIPTION
**Pull request by TheEssem**
Just a small edit to `AudioPlayerConfiguration` that adds the ability to add new `MediaContainerProbe` instances. This is useful for adding support for new HTTP/local file formats or adding a new source that depends on some sort of weird codec.

### CODE REVIEW

File: LavalinkServer/src/main/java/lavalink/server/config/AudioPlayerConfiguration.kt [hunk 1]
```diff
@@ -109,7 +114,7 @@ class AudioPlayerConfiguration {
         if (sources.isTwitch) audioPlayerManager.registerSourceManager(TwitchStreamAudioSourceManager())
         if (sources.isVimeo) audioPlayerManager.registerSourceManager(VimeoAudioSourceManager())
         if (sources.isMixer) audioPlayerManager.registerSourceManager(BeamAudioSourceManager())
-        if (sources.isLocal) audioPlayerManager.registerSourceManager(LocalAudioSourceManager())
+        if (sources.isLocal) audioPlayerManager.registerSourceManager(LocalAudioSourceManager(mcr))
```
Comment (freyacodes) [inline LavalinkServer/src/main/java/lavalink/server/config/AudioPlayerConfiguration.kt:117]: Is this additive? Are the default probes removed if the collection is empty?

Comment (viztea) [reply LavalinkServer/src/main/java/lavalink/server/config/AudioPlayerConfiguration.kt:117]: > Is this additive?

Nope

Comment (TheEssem) [reply LavalinkServer/src/main/java/lavalink/server/config/AudioPlayerConfiguration.kt:117]: No, LocalAudioSourceManager when initialized without any arguments uses the default probe list and `MediaContainerRegistry#extended()` simply adds onto the default probe list.

Comment (freyacodes) [reply LavalinkServer/src/main/java/lavalink/server/config/AudioPlayerConfiguration.kt:117]: Ah yes, I see

**Review by freyacodes — CHANGES_REQUESTED** [review]
This feature needs to be documented in PLUGINS.md otherwise plugin developers have no chance of knowing about it

**Comment by TheEssem** [comment]
Sorry about that, not sure how it closed itself.

**Review by freyacodes — APPROVED** [review]
Thanks! Sorry for the wait

**Pull request merged**